src/shared/common/components/case-notes.jsx

```diff
@@ -41,7 +41,7 @@ AuditEvent.propTypes = {
 };
 
 const TimelineItem = (refreshNotes) => ({ type, body, title, timelineItemUUID }) => {
-    const isCaseNote = ['MANUAL', 'ALLOCATE', 'CLOSE', 'REJECT', 'PHONECALL', 'REQUEST_CONTRIBUTION', 'FOLLOW_UP', 'FOLLOW_UP_NOT_COMPLETED', 'WITHDRAW'].includes(type);
+    const isCaseNote = ['MANUAL', 'ALLOCATE', 'CLOSE', 'REJECT', 'PHONECALL', 'REQUEST_CONTRIBUTION', 'SEND_TO_WORKFLOW_MANAGER', 'FOLLOW_UP', 'FOLLOW_UP_NOT_COMPLETED', 'WITHDRAW'].includes(type);
```

Comment:
At some point here maybe it will be better to figure out what all non-case-note types we have and invert this check.
This would reduce the need for keeping and maintaining this list. Not a must do for this change as you're just following existing pattern :)

Reply:
I'd also try and find a wait of making these into some sort of enum, I really do not like the use of hardcoded strings like this when we might need to use them elsewhere in the future...